src/renderer/renderScene.ts

```diff
@@ -51,46 +51,11 @@ export function renderScene(
 
   const context = canvas.getContext("2d")!;
 
-  // Get initial scale transform as reference for later usage
-  const initialContextTransform = context.getTransform();
```

Comment:
This was mostly used for getting the pixel ratio of the device. The canvas is scaled outside of this function